recipes/mapio/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "MapIO" %}
+{% set version = "0.6.1" %}
+{% set sha256 = "c05fe8ae74253decc2fdb0b4129743e119db4594cc6f1bb15fc5b2049f8a8b36" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/usgs/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [not py35]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    # When setuptools is available add the `--single-version-externally-managed --record record.txt` above.
+    - setuptools
+    # if your project compiles code (such as a C extension) then add `toolchain` as a build requirement.
+  run:
+    - python
+    - numpy 1.12.*
```

Comment:
Unless you know that the library only works with these exact versions, please remove the version pins. The problem with being too specific is that the it rapidly becomes impossible to maintain the package or for users to install it. 

If you need recent versions, e.g. numpy 1.12 or later, then a version spectrum like `numpy >=1.12` will do what you want.

Also, does this depend on `openquake`? If so, and if it is not on conda-forge, then you will need to add a recipe for that to.